electron-app/src/plugins/vuetify.ts

```diff
@@ -17,7 +17,17 @@ export default new Vuetify({
         error: '#FF5252',
         info: '#2196F3',
         success: '#4CAF50',
-        warning: '#FFC107'
+        warning: '#FFC107',
+        rotkigreen: '#06D6A0',
+        rotkired: '#F03A47',
+        rotkiorange: '#E96930',
+        rotkiblack: '#0A0908',
+        rotkibeige: '#F2EFEA',
```

Comment:
And here I was wondering where these colors came from. This is another thing I was not aware of.